beacon/sync/src/main/java/tech/pegasys/teku/beacon/sync/forward/multipeer/chains/ThrottlingSyncSource.java

```diff
@@ -31,23 +33,29 @@
 
 public class ThrottlingSyncSource implements SyncSource {
   private static final Logger LOG = LogManager.getLogger();
+
+  // 1 minute
```

Comment:
i'd probably prefer the const was 'time_out_seconds' than a floating comment like this...